testsuite/tests/kuadrant/reconciliation/cr_configuration/test_reconcile_limitador.py

```diff
@@ -0,0 +1,64 @@
+"""Tests for Kuadrant sub component - Limitador CR configuration via Kuadrant CR"""
+
+import pytest
+
+pytestmark = [pytest.mark.kuadrant_only]
+
+
+@pytest.fixture()
+def commit():
+    """Omitting authorization and rate limiting as it is not needed"""
+
+
+@pytest.fixture(autouse=True)
+def kuadrant_clean_up(request, kuadrant):
+    """
+    Return fields to default values.
+    This can be simplified once https://github.com/Kuadrant/kuadrant-operator/issues/617 is fixed.
+    """
+
+    def _finalize():
+        kuadrant.limitador = {"replicas": 1, "resourceRequirements": {"requests": {"cpu": "250m", "memory": "32Mi"}}}
+
+    request.addfinalizer(_finalize)
+
+
+@pytest.mark.parametrize(
+    "field, value",
+    [
+        pytest.param("replicas", 2, id="replicas"),
+        pytest.param(
+            "resourceRequirements", {"requests": {"cpu": "300m", "memory": "40Mi"}}, id="resourceRequirements"
+        ),
+    ],
+)
+def test_fields_to_reconcile(kuadrant, field, value):
+    """
+    Test:
+        - assert that Limitador deployment replicas is equal to 1
+        - change replicas field to 2 in Kuadrant CR
+        - assert that Kuadrant CR Limitador replicas is equal to 2
+        - assert that Limitador deployment replicas is equal to 2
+    """
+    kuadrant.limitador = {field: value}
+
+    assert value == kuadrant.limitador[field]
+    assert str(value) in str(kuadrant.limitador_deployment.model.spec)
```

Comment:
will this work?
```suggestion
    assert getattr(kuadrant.limitador_deployment.model.spec, field) == value
```

Reply:
Great idea!

Reply:
This is not working for `resourceRequirements` as they are buried deep down in deployment yaml (`kuadrant.limitador_deployment.model.spec.template.spec.volumes ...`)